pom.xml

```diff
@@ -431,35 +470,28 @@
 			<activation>
 				<jdk>1.8</jdk>
 			</activation>
+			<properties>
+				<javac.version>9+181-r4173-1</javac.version>
```

Comment:
uh this is oddly specific. Do we have to do this?

Reply:
I believe this was in the errorprone docs.